Dockerfile

```diff
@@ -37,7 +37,7 @@ FROM alpine as hydra-tui
 RUN apk update && apk add ncurses-terminfo
 COPY --from=build-hydra-tui /build/hydra-tui-result/bin/hydra-tui /bin/
 STOPSIGNAL SIGINT
-ENTRYPOINT ["sh", "-c", "TERMINFO=/usr/share/terminfo hydra-tui $@"]
+ENTRYPOINT ["sh", "-c", "TERMINFO=/usr/share/terminfo hydra-tui $@", "--"]
```

Comment:
Are you sure this goes this way around? I would have expected `"TERMINFO=/usr/share/terminfo hydra-tui -- $@"`?

Reply:
You're right... and still it works... give me a sec

Reply:
No, my change is actually correct. It does something like that:

```bash 
docker run -it hydra-tui -n 2
```

Becomes:

```bash 
sh -c "TERMINFO=/usr/share/terminfo hydra-tui $@" -- -n 2
```

And that is actually what we want: `--` is sent to `sh` (`-n` and `2` are too actually) so `sh` decides that all the remaining parameters are to be sent to the command and so it does send them to the command. Doing so, it replaces `$@` with the remaining parameters.